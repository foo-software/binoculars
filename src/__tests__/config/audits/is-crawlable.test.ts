import auditIsCrawlable from '../../../config/audits/is-crawlable';

describe('is-crawlable audit', () => {
  it('should match snapshot', () => {
    expect(auditIsCrawlable('en')).toMatchSnapshot();
  });
});
