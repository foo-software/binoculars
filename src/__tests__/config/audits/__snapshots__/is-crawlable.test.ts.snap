// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is-crawlable audit should match snapshot 1`] = `
Object {
  "description": "Search engines are unable to include your pages in search results if they don't have permission to crawl them. [Learn more](https://web.dev/is-crawable/).",
  "title": "Page isnâ€™t blocked from indexing",
}
`;
